fastestimator/dataset/dataloader.py

```
@@ -137,7 +138,10 @@ def shutdown(self) -> None:
         the test suites.
         """
         if isinstance(self._iterator, _MultiProcessingDataLoaderIter):
-            self._iterator._shutdown_workers()
+            try:
+                self._iterator._shutdown_workers()
+            except:
```

Ruff

Do not use bare `except`

(E722)